gokart/testing/pandas_assert.py

```diff
@@ -0,0 +1,28 @@
+import pandas as pd
+
+
+def assert_frame_equal(actual: pd.DataFrame, expected: pd.DataFrame, **kwargs):
+    """
+    Assert that two DataFrames are equal.
+    Note 
```

Comment:
There's no contents of `Note`.

Reply:
It needs a description of difference of `pd.testing.assert_frame_equal`, IMO

Reply:
@hirosassa @Hi-king 
I've added a description of difference between pd.testing.assert_frame_equal.

Please re-review!

Reply:
👍